src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -10,7 +10,7 @@
   "defaultName": "WebApplication",
   "description": "A project template for creating a Blazor app that runs on WebAssembly and is optionally hosted by an ASP.NET Core app. This template can be used for web apps with rich dynamic user interfaces (UIs).",
   "groupIdentity": "Microsoft.Web.Blazor.Wasm",
-  "precedence": "6002",
+  "precedence": "7002",
```

Comment:
It seems like this might be a good time to normalize them all, as long as this is > then the 5.0 version or any conceivable future SxS servicing value, this should be good.